chapters/3-measurement.tex

```diff
@@ -291,20 +338,24 @@ \subsection{Data flowcharts}
 such as panel datasets like ``person-year'' structures
 and multi-level data like ``district-school-teacher-student'' structures.
 
-\section{Expressing research design in data maps}
+\section{Research designs and data map needs}
 
-After you have set up your data map,
+After you have made the first draft of your data map,
 you need to carefully think about your research design
-and which research variables you will need in the data analysis
+and decide which research design variables you will need in the data analysis
 to infer the relation between differences in measurement variables
-and your research design.
+and your research design.\index{research design variables}
```

Comment:
i wonder about the sequencing here. to me this seems like it should come before or at least be part of the process of creating a data map; part of the understanding of how you set up your data. in practice i think all i suggest changing is to say "When drafting your data map, you need to ..." instead of "After you have made the first draft of your data map, you need to ..."

Reply:
I think a project team should cover the content in these two sections at the same time, but in the book it is separate sections. Similarly to data visualization, we talk about it during research outputs but it can be used already from the data cleaning stage. 

I like your suggestion, and if you think that is sufficient, then I am happy to just accept that.

Reply:
Yep, i think just changing the wording as i suggested is sufficient.

Reply:
I am done quickly going over your edits. You went for a third option in your edits. I think what you committed also works.